pkg/csi/service/wcp/controller.go

```diff
@@ -253,39 +252,31 @@ func (c *controller) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequ
 			}
 			accessibleNodes = append(accessibleNodes, overlappingNodes...)
 			log.Infof("Storage pool Accessible nodes for volume topology: %+v", accessibleNodes)
-		} else if param == common.AttributeHostLocal {
```

Comment:
If this mode is being deprecated, please also make sure that the FVT tests that exercise this mode are removed.

Reply:
Mansi told me that FVT does not have any tests specific to this.

Reply:
Also this is not a "mode" that FVT used. This was just the old way of identifying a storage class to be vsan sna storage class. So in fact what we need to do is stop sending this param down to CSI from external provisioner now. @Palllabi can you please make sure to remove the code in external provisioner that sends this AttributeHostLocal down to CSI.